File: tests/rspec/pages/channel/field_groups.rb
class FieldGroups < ControlPanelPage
  element :create_new, '.sidebar a.btn.left'
  elements :field_groups, '.folder-list > li'
  elements :field_groups_edit, '.folder-list li.edit a'
  elements :field_groups_fields, '.folder-list > li > a'

  def load
    visit '/admin.php?/cp/fields'
  end
end

File: tests/cypress/cypress/elements/pages/channel/FieldGroups.js
import ControlPanel from '../ControlPanel'
import FieldGroupForm from './FieldGroupForm'
const form = new FieldGroupForm

class FieldGroups extends ControlPanel {
    constructor() {
        super()
        this.url = 'admin.php?/cp/fields';

        this.selectors = Object.assign(this.selectors, {
            //"create_new": '.sidebar a.button--action.left',
            "create_new": 'a[href="admin.php?/cp/fields/groups/create"]', //added /groups/

            "field_groups": '.folder-list > div',
            //"field_groups_edit": '.folder-list div li.edit a',
            "field_groups_edit": 'a[title="Edit"]',

            "field_groups_fields": '.folder-list > div > a',
        })
    }

    change(name){
        form.get('new_name').clear().type(name)
        form.get('submit').first().click()
    }

    save_field_group(name) {
        form.get('name').clear().type(name)
        form.get('submit').first().click()
    }

}
export default FieldGroups;